README.md

```diff
@@ -256,7 +256,9 @@ edit its security group rules to only allow access to VMs in the
 ### GitHub settings
 
 * `GITHUB_ACCESS_TOKEN`: Your GitHub access token (required). Get it from
-  https://github.com/settings/tokens
+  https://github.com/settings/tokens, and enable the `read:user` and
+  `read:user` scopes on the token.  The user must also be a member of
```

Comment:
@itsjeyd Oops :)  Thanks for noticing that!  Fixed with https://github.com/open-craft/opencraft/pull/220/commits/5b239bdf05d192034b9ba99354275d7219b35d8a#diff-04c6e90faac2675aa89e2176d2eec7d8R259.